eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyCachingTest.java

```diff
@@ -59,4 +62,26 @@ void putsObjectToLocalCache(@TempDir final Path path) throws Exception {
                 .exists()
         );
     }
+
+    @Test
+    void checksPresenceOfObject(@TempDir final Path path) throws Exception {
+        final String content = "[] > main\n";
+        new LengthOf(
+            new TeeInput(
+                new InputOf(content),
+                new OutputTo(
+                    path.resolve("pulled/master/org/example/main.eo")
+                )
+            )
+        ).value();
+        final Objectionary objectionary = new OyCaching(
```

Comment:
@MikhailLipanin you can inline `objectionary` variable

Reply:
done